webpack/assets/javascripts/react_app/components/MemoryAllocationInput/MemoryAllocationInput.js

```diff
@@ -107,13 +108,14 @@ const MemoryAllocationInput = ({
         <Col>
           <NumericInput
             value={value}
+            id={id}
             format={format}
             parser={parser}
             onChange={handleChange}
             label={label}
-            name={name}
             disabled={disabled}
           />
+          <input type="hidden" name={name} value={value * MEGABYTES} />
```

Comment:
Nice addition!
though for other usages, and make it transparent to the consumer,  how about extending the ` NumericInput` component to have this hidden input?
i.e
```js
const ForemanNumericInput = ({ name, value }, ...otherProps) => (
<>
  <NumericInput value={value} {...otherProps} />
  <input type="hidden" name={name} value={value * MEGABYTES} />
</>  
)


```

Reply:
I am not completely sure I understand this request. The hidden input is not relevant for the general NumericInput but only to the MemoryAllocationInput. Do you still think it should be an extension to the NumericInput?

Reply:
AFAIK the `NumericInput` has an issue with the `value` prop (it adds the formatting result to it), so extending this component with a hidden input for the value would be beneficial to other consumers.

Reply:
I strongly belive this has nothing to do with other components, we just want the real value of the input to be something else, then the input shows and that's not what NumericInput is meant to do. Lets keep this 'weird' behaviour contained :)
It looks much cleaner this way :+1: